plugin/__init__.py

```diff
@@ -0,0 +1,39 @@
+from pyramid.events import NewRequest
+from kinto.core.events import AfterResourceChanged
+from pywebpush import WebPusher
+
+def on_new_request(event):
+    cache = event.request.registry.cache
+    storage = event.request.registry.storage
+
+    subsID = req.param('subsID')
```

Comment:
We will not read this form the querystring but from the body.
I don't think we will handle adding subscription for each new request but probably on the Service endpoint that we will create for it.